packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -537,12 +538,16 @@ const helperBehavior = {
           // it. We could hold it until it crosses some threshold, then sell it
           // to the AMM, or we could transfer it to the reserve. At least it's
           // visible in the accounting.
-          vaultSeat.decrementBy(
-            state.retainedCollateralSeat.incrementBy({
```

Comment:
Note that the original here was written backward: with the incrementBy inside the decrementBy, violating our normal convention. No such confusion in the new code.